next-desolates-map/components/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,466 @@
+                        <li className="">
+                            <div className="min-h-0 lg:ml-3 h-8 w-full bg-black bg-opacity-75 rounded-lg outline-cool mb-5 lg:mb-1">
+                                    
+                                    <input
+                                        id="search-input"
+                                        type="text"
+                                        name="planet-no"
+                                        size={40}
+                                        className="w-full py-1 pl-4 text-lg text-left bg-transparent rounded-r-lg text-primary font-cool placeholder-primary focus:outline-none"
+                                        placeholder="Enter planet name..."
+                                        onChange={handleInputChange}
+                                        autoComplete="off"
+                                        onKeyDown={handleKeyDown}
+                                    >
+                                    </input>
+                                </div>
+                        </li>
+
+                        {/* Planet ID */}
+                        <li className="">
+                            <div className="min-h-0 lg:ml-10 h-8 flex items-center">
+                                <p className="text-lg font-cool text-primary">123</p>
+                            </div>
+                        </li>
+
+                        {/* Planet Input */}
+                        <li className="">
+                            <div className="min-h-0 lg:ml-3 h-8 w-full bg-black bg-opacity-75 rounded-lg outline-cool mb-5 lg:mb-1">
+                                    <input
+                                        id="search-input"
+                                        type="text"
+                                        name="planet-no"
+                                        size={40}
+                                        className="w-full py-1 pl-4 text-lg text-left bg-transparent rounded-r-lg text-primary font-cool placeholder-primary focus:outline-none"
+                                        placeholder="Enter planet name..."
+                                        onChange={handleInputChange}
+                                        autoComplete="off"
+                                        onKeyDown={handleKeyDown}
+                                    >
+                                    </input>
+                                </div>
+                        </li>                        
+
+                        </ul>
+
+                    {/* Cancel & Save btns */}
+
+                    <div className="min-h-0 max-h-screen  h-8 flex justify-end">
+                        <button
+                            className={`font-cool p-2 mr-3 text-sm text-center text-primary transition-colors duration-200 rounded-lg hover:text-white hover:bg-primary w-20 outline-cool backdrop-filter backdrop-blur bg-faded`}
```

Comment:
You shoudl use the double quotes **"** instead of **{``}**

Reply:
Done.